Runtime/Components/Cinemachine3rdPersonFollow.cs

```diff
@@ -121,12 +140,12 @@ public override void MutateCameraState(ref CameraState curState, float deltaTime
 
         void PositionCamera(ref CameraState curState, float deltaTime)
         {
-            var targetPos = FollowTargetPosition;
-            var prevTargetPos = deltaTime >= 0 ? PreviousFollowTargetPosition : targetPos;
+            var followTarget = FollowTargetPosition;
+            var prevTargetPos = deltaTime >= 0 ? m_PreviousFollowTargetPosition : followTarget;
```

Comment:
all the "prev" states need to be reset when deltaTime < 0, including the new ones just added